staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/interfaces.go

```diff
@@ -60,6 +60,6 @@ type WebhookInvocation struct {
 
 // Dispatcher dispatches webhook call to a list of webhooks with admission attributes as argument.
 type Dispatcher interface {
-	// Dispatch a request to the webhooks using the given webhooks. A non-nil error means the request is rejected.
-	Dispatch(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces, hooks []*WebhookInvocation) error
+	// Dispatch a request to the webhooks. Dispatcher may choose not to call a hook. A non-nil error means the request is rejected.
```

Comment:
this needs to be crisper about specifically why the webhook can choose not to call the hook